Looms.PoS/Looms.PoS.Application/Features/Refund/Queries/GetRefunds/GetRefundsQuery.cs

```diff
@@ -0,0 +1,8 @@
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Looms.PoS.Application.Features.Refund.Queries.GetRefunds;
+
+public record GetRefundsQuery : IRequest<IActionResult>
```

Comment:
Inherit LoomHttpRequest, will be needed later with permissions

Reply:
added LoomHttpRequest